README.rst

```diff
@@ -29,13 +29,23 @@ Setting up your environment
 This package uses Poetry to manage Python dependencies and CSS and JS assets are
 built using a number of Node dependencies.
 
-You will need to specify the ``FONTAWESOME_NPM_AUTH_TOKEN`` environment variable
-before installing Node dependenvies. You should be using ``direnv`` locally for
-secret management and encryption. Ask for this token, it's private.
+You will need the following requirements:
```

Comment:
This change is related to #172, the token is not required because we're using a hosted kit.